Automattic/wp-calypso
# VideoPress: Update media library preview to use new player

This PR updates the media library preview area to use the new VideoPress player, powered by videojs:

<img width="1033" alt="Screen Shot 2021-02-09 at 1 37 34 PM" src="https://user-images.githubusercontent.com/789137/107433116-26bee880-6ae6-11eb-8ec0-fc0569be7126.png">

I've tweaked the function that fetches the video info while proxied to support what the new player expects.

**To Test**
 * Load up a VideoPress video in your media library.
 * The video should load and work as expected.
 * Drag a new video into the media library. After it uploads, edit it and you should see the conversion screen. When it completes the player should refresh.
 * Load up a private site and verify that you can view/play a video as long as you are a member of the site.
